packages/proxy/src/providers/anthropic.ts

```diff
@@ -332,6 +341,7 @@ export function anthropicEventToOpenAIEvent(
 export function anthropicCompletionToOpenAICompletion(
   completion: AnthropicCompletion,
   isFunction: boolean,
+  isStructuredOutput: boolean,
 ): ChatCompletion {
   console.log("COMPLETION", JSON.stringify(completion, null, 2));
```

Comment:
mind removing this console.log statement

Reply:
Done